packages/gestalt/src/TableRowExpandable.js

```diff
@@ -35,6 +44,9 @@ export default function TableRowExpandable(props: Props): Node {
   const [expanded, setExpanded] = useState(false);
   const hoverStyle = props.hoverStyle || 'gray';
   const cs = hoverStyle === 'gray' ? cx(styles.hoverShadeGray) : null;
+  const { stickyColumns } = useContext(TableContext);
+  const rowRef = React.useRef();
+  const [columnWidths, setColumnWidths] = useState([]);
 
   const handleButtonClick = ({ event }) => {
     setExpanded(!expanded);
```

Comment:
This always makes me nervous…we should use functional updates for this sort of toggling to prevent using stale values: https://reactjs.org/docs/hooks-reference.html#functional-updates

Reply:
Thanks for pointing to this... I feel we do that across many cmps.

Reply:
Since it's unrelated to these changes, I'll make a separate ticket to look at this across our cmps